Code/Editor/Objects/ObjectManager.cpp

```diff
@@ -1434,7 +1343,7 @@ void CObjectManager::EndEditParams([[maybe_unused]] int flags)
 {
     m_bSingleSelection = false;
     m_currEditObject = nullptr;
-    //m_bSelectionChanged = false; // don't need to clear for ungroup
+    // m_bSelectionChanged = false; // don't need to clear for ungroup
```

Comment:
nit: remove commented out code?

Reply:
Oh crap I accidentally ran clang-format on that file 🙈 I'll revert those changes and re-upload sorry 😩